Twig/Extension.php

```diff
@@ -59,6 +65,18 @@ public function piwikCode()
             return '<!-- Piwik is disabled due to webfactory_piwik.disabled=true in your configuration -->';
         }
 
+        if ($this->disableCookies === null) {
+            trigger_error('The "disableCookies" configuration key is missing. Please define the "disableCookies" key explicit. In newer versions this will be "true" by default.', E_USER_DEPRECATED);
```

Comment:
Deprecation notices need to be `@`-silenced, `@trigger_error('...', E_USER_DEPRECATED)`